### DISCUSSION TITLE
Alerts not triggering webhook notifications to MSTeams, nor Slack

### DISCUSSION THREAD

**Discussion by dom-advancedcommerce** [datalust/seq-tickets#2323] [Help]
We are trying to leverage some of the reporting to alert us when certain counters hit thresholds.

We have added the MSTeams & Slack apps into the SEQ instance, all configured.

MSTeams & Slack are configured to receive the notifications.

We are using the same MSTeams & Slack channel for all incomming events.

SEQ is flagging when the thresholds are being hit which should then trigger these webhook notifications, however, of the alerts we have in place (7), only 1 sends a message which hits the MSTeams channel.

If we switch the alerts to Slack, nothing comes through. 

Alerting using SEQ is going to be an important factor for our realtime DEVOPS support. If we can't use it, we will have to find an alternative to SEQ.

What can we do to try and trace where and why we have this issue.

**Comment by nblumhardt** [comment]
Hi Dom,

In the Events screen, if you pick a random event and choose "Send to app > (Your Teams app instance)" does the event reach Teams? And does this work for the Slack case?

Things to check:

 * In Settings > Apps, beside each app instance name will be a clickable "lightning bolt" hyperlink; selecting this will show any errors produced by the app
 * In the app instance itself, if you're using it for alerting, make sure the "Stream incoming events" checkbox is unchecked
 * Are your Slack and Teams plug-ins up-to-date? (Checkable in Settings > Apps)

If none of the above helps, sending these things to our support address (redacted if necessary) will help track it down:

 * Screenshots of all of the configuration involved - one of the problem alerts, one the app instances
 * Screenshot of the Alerts screen showing one of the "fired" alerts
 * A diagnostic report from Settings > Diagnostics

Hope this helps!

Nick

**Reply by dom-advancedcommerce** [reply]
HI @nblumhardt 

Email sent with requested information

**Reply by nblumhardt** [reply]
Thanks Dom, we'll take a look 👍
